api/src/main/java/org/pmiops/workbench/utils/CohortMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.utils;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import org.pmiops.workbench.db.model.DbCohort;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.Cohort;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class},
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

Comment:
This was necessary because DbCohort and Cohort each have fields the other lacks.

Reply:
replaced with explicit ignore rules for each one.